wsd/wopi/WopiStorage.cpp

```diff
@@ -788,10 +784,12 @@ void WopiStorage::uploadLocalFileToStorageAsync(const Authorization& auth, LockC
         LOG_DBG(wopiLog << " async upload request: " << httpRequest.header().toString());
 
         // Make the request.
-        _uploadHttpSession->asyncRequest(httpRequest, socketPoll);
+        if (_uploadHttpSession->asyncRequest(httpRequest, socketPoll))
```

Comment:
Interestingly my build fails because of this statement because this function returns void! (i.e: void http::Session::asyncRequest)

Reply:
Had same issue, hence I fixed it - pending review.

Resolved w/ https://github.com/CollaboraOnline/online/pull/9839
commit https://github.com/CollaboraOnline/online/pull/9839/commits/34ec1db7f8a745279db8027de527b595bbd01eb8 

@caolanm @lpranam